CondFormats/HcalObjects/interface/HcalCondObjectContainer.h

```diff
@@ -108,15 +108,28 @@ HcalCondObjectContainer<Item>::initContainer(DetId fId) {
 
   if (fId.det()==DetId::Hcal) {
     switch (HcalSubdetector(fId.subdetId())) {
-    case(HcalBarrel) : for (unsigned int i=0; i<sizeFor(fId); i++) HBcontainer.push_back(emptyItem); break;
-    case(HcalEndcap) : for (unsigned int i=0; i<sizeFor(fId); i++) HEcontainer.push_back(emptyItem); break;
-    case(HcalOuter) : for (unsigned int i=0; i<sizeFor(fId); i++) HOcontainer.push_back(emptyItem); break;
-    case(HcalForward) : for (unsigned int i=0; i<sizeFor(fId); i++) HFcontainer.push_back(emptyItem); break;
-    case(HcalTriggerTower) : for (unsigned int i=0; i<sizeFor(fId); i++) HTcontainer.push_back(emptyItem); break;
-    case(HcalOther) : if (extractOther(fId)==HcalCalibration) 
-	for (unsigned int i=0; i<sizeFor(fId); i++) CALIBcontainer.push_back(emptyItem); break;
-      break; 
-    default: break;
+      case HcalBarrel:
+	for (unsigned int i = 0; i < sizeFor(fId); i++) HBcontainer.push_back(emptyItem);
+	break;
+      case HcalEndcap:
+	for (unsigned int i = 0; i < sizeFor(fId); i++) HEcontainer.push_back(emptyItem);
+	break;
+      case HcalOuter:
+	for (unsigned int i = 0; i < sizeFor(fId); i++) HOcontainer.push_back(emptyItem);
+	break;
+      case HcalForward:
+	for (unsigned int i = 0; i < sizeFor(fId); i++) HFcontainer.push_back(emptyItem);
+	break;
+      case HcalTriggerTower:
+	for (unsigned int i = 0; i < sizeFor(fId); i++) HTcontainer.push_back(emptyItem);
+	break;
+      case HcalOther:
+	if (extractOther(fId) == HcalCalibration) { 
+	  for (unsigned int i = 0; i < sizeFor(fId); i++) CALIBcontainer.push_back(emptyItem);
+	}
+	break; 
+      default:
+	break;
     }

```

Comment:
@davidlt Since this is about style: Does 1 space of indentation really help to guide the eye?